WcaOnRails/app/models/competition.rb

```diff
@@ -156,6 +156,59 @@ def user_cannot_demote_themself
     end
   end
 
+  attr_accessor :receive_registration_emails
+  def receive_registration_emails=(r)
+    @receive_registration_emails = ActiveRecord::Type::Boolean.new.type_cast_from_database(r)
```

Comment:
How does this sound?

```
receive_registration_emails comes from a client side checkbox input. For database fields, 
ActiveRecord  does the conversion from "1" and "0" to true and false, but since 
`receive_registration_emails` is not an actual database field, ActiveRecord doesn't
help us here.
```